2-mandatory/4-people-i-know.js

```diff
@@ -378,13 +378,23 @@ First, I want you to find all of my friends who are 35 or older.
 */
 
 let thirtyFiveOrOlder = [];
+for (let friend of friends){
+  if (friend.age >= 35){
+    thirtyFiveOrOlder.push(friend);
+  }
+}
 
 /*
 3) Find the email address
 Next, I want you to find all of my friends who work for "POWERNET" and then store their emails in the array below
 */
 
 let powerNetEmails = [];
+for (let friend of friends) {
+  if (friend.company=="POWERNET") {
```

Comment:
try to use `===` instead of `==` in almost all cases; `==` will do weird things like saying `"2" == 2` is `true` despite one being a string and another being a number